cmd/kyverno/main.go

```diff
@@ -368,6 +368,11 @@ func main() {
 	registerWrapperRetry := common.RetryFunc(time.Second, webhookRegistrationTimeout, webhookCfg.Register, "failed to register webhook", setupLog)
 	registerWebhookConfigurations := func() {
 		certManager.InitTLSPemPair()
+		pInformer.WaitForCacheSync(stopCh)
+		kubeInformer.WaitForCacheSync(stopCh)
+		kubeKyvernoInformer.WaitForCacheSync(stopCh)
+		kubedynamicInformer.WaitForCacheSync(stopCh)
```

Comment:
We no longer use dynamic informers, do we need to wait for caches?

Reply:
I removed them.